### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MarinaT-N**
i really want to get feedback about all my mistakes and how I could improve my test.Thanks

### CODE REVIEW

File: cypress/e2e/registration.cy.js [hunk 1]
```diff
@@ -1,11 +1,90 @@
...
+    cy.get(':nth-child(4) > .group-header > .header-wrapper').click() //click left sidebar
+
+    cy.get('#state').click().type(`${state}`+'{enter}');  
+    cy.get('#city').click().type(`${city1}`+'{enter}');
+
+    cy.get('#submit').scrollIntoView();  
+    cy.get('#submit').click(); 
 
-  it('', () => {
+    //TEST
```
Comment (VadimMentorQA) [inline cypress/e2e/registration.cy.js:58]: Do we need this comment?

File: cypress/e2e/registration.cy.js [hunk 2]
```diff
@@ -1,11 +1,90 @@
...
+        .should('contain.text',`${address}`);
+
+    cy.get('tbody > :nth-child(10) > :nth-child(2)')
+        .should('contain.text',`${state} ${city1}`);  
   });
 });
+
+
+
+
```
Comment (VadimMentorQA) [inline cypress/e2e/registration.cy.js:90]: Delete empty rows

File: cypress/e2e/registration.cy.js [hunk 3]
```diff
@@ -1,11 +1,90 @@
 /// <reference types='cypress' />
+const { generateRegistration } = require("../support/generation");
```
Comment (VadimMentorQA) [inline cypress/e2e/registration.cy.js:2]: Add this file, please.

**Review by VadimMentorQA — CHANGES_REQUESTED** [review]
Check comments, please. 
Make sure you have added all required files.

**Review by NikitaZH1812 — APPROVED** [review]

**Pull request closed without merging**